packages/host/app/templates/card.hbs

```diff
@@ -5,6 +5,8 @@
 <Editor::CodeLink />
 
 {{on-key 'Ctrl+.' this.toggleOperatorMode}}
+{{!-- Ctrl+. doesn't work in ubuntu --}}
```

Comment:
Interesting. I was curious and it looks it's because of this https://superuser.com/questions/1672408/linux-gnome-how-to-disable-ctrl-shortcut

Reply:
lol yeah, ctrl+. brings up my emoji keyboard and it seems to have a higher precedence than any of the events in the browser